test/cql/vs_cdc_enable_disable_test.cql

```diff
@@ -2,10 +2,9 @@
 -- do not depend on how stream IDs are assigned to partition keys.
 
 -- Error messages contain a keyspace name. Make the output stable.
--- CDC and tablets are not working together yet, turn them off.
 CREATE KEYSPACE ks
     WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1} AND
-    tablets = {'enabled': false};
+    tablets = {'enabled': true};
```

Comment:
Again, it's not clear to me why a patch about vector search wants to modify a CDC test.
Here I'm less sure what is the right thing to do - it seems the existing test **forces** vnodes (it's not a configuration option) which isn't good, but I'm not sure that forcing it to be tablets is good either.
It seems to me that the more correct change is to remove the "tablets = ..." option entirely from this test. The test will then hopefully use some sort of global default, which different CI jobs can set to either be vnodes or tablets?